code/modules/cargo/exports/materials.dm

```diff
@@ -107,3 +107,9 @@
 	message = "cm3 of Hot Ice"
 	material_id = /datum/material/hot_ice
 	export_types = /obj/item/stack/sheet/hot_ice
+
+/datum/export/material/metal_hydrogen
+	cost = 550
```

Comment:
why is it worth so much

Reply:
is worth a lot because is hard to make (a normal atmos team of 2/3 people can make 30 or 40 sheets will take over 50 minutes to make) since you need:
- make bz farm 
- make electolyzer (wait for science and mining to do their thing)
- burn plasma with low o2 to make h2o and use electrolizer to make h2
- burn plasma with a lot of o2 to make trit
- combine h2, trit and plasma and start fusion
- use bz and a lot of h2 in a spaced chamber and heat up the gases to over 1e6 K and 1e6 kpa
- use 30 sheets to make golem shell and 10 more to activate the shell

Reply:
alright, sounds fine to me